actions/add_games.rb

```diff
@@ -0,0 +1,27 @@
+require './classes/game'
+require './inputs/item_inputs'
+require './inputs/game_inputs'
+require_relative 'instance_creator'
+require 'json'
+
+class AddGame
+  def self.add_new_game(games)
+    item_name, item_genre, item_author_first, item_author_last, item_label_title,
+    item_label_color, item_date = ItemInputs.item_inputs
+    multiplayer, game_last_played = GameInput.game_inputs
+
+    game_instance = Game.new(name: item_name, multiplayer: multiplayer == 'y', last_played_at: game_last_played,
+                             genre: {}, author: {}, label: {}, publish_date: item_date)
+
+    genre = InstanceCreator.genre_instance(item_genre, game_instance)
+    author = InstanceCreator.author_instance(item_author_first, item_author_last, game_instance)
+    label = InstanceCreator.label_instance(item_label_title, item_label_color, game_instance)
+
+    game = { 'json_class' => 'game', 'title' => game_instance.name, 'genre' => genre,
+             'author' => author, 'label' => label, 'publish_date' => game_instance.publish_date,
+             'multiplayer' => game_instance.multiplayer, 'last_played_at' => game_instance.last_played_at }
```

Comment:
- [ ] Let's specify the type of input and format we want the user to add as this is supposed to be a date input to avoid the user putting any type of data. Also, let's make calculations in the `Game` class on how to get the `can_be_archived?` if the parent's method returns true AND if last_played_at is older than 2 years because `can_be_archived?` is meant to return true or false.

<img width="235" alt="Loom_a4VYdMiJFF" src="https://user-images.githubusercontent.com/81468675/153515482-47d61dff-348a-4277-8fc7-ced0064dd8cc.png">